daemon/endpoint.go

```diff
@@ -1020,10 +1021,14 @@ type endpointManager struct {
 // * After, K8sWatcher
 // * Later, IPAM.
 func NewEndpointManager(d *Daemon, epSync *endpointsynchronizer.EndpointSynchronizer) *endpointManager {
-	return &endpointManager{
-		d:               d,
-		EndpointManager: endpointmanager.NewEndpointManager(epSync),
-	}
+	self := &endpointManager{d: d}
+	mgr := endpointmanager.NewEndpointManager(epSync)
+	epGC := option.Config.EndpointGCInterval
+	if epGC > 0 {
```

Comment:
Style nit: we can fold this into this if-statement

Reply:
hehe yeah my oldschool desire for <80 character lines ended up with this, it's not strictly necessary.